app/test-utils/helper.js

```diff
@@ -21,5 +21,18 @@ export const getCsrfTokenFromGet = async (app, getPath) => {
       cookies = getRes.headers['set-cookie'];
       csrfToken = extractCsrfToken(getRes);
     });
-  return { cookies, csrfToken };
+
+  return { cookies, csrfToken, app };
+};
+
+export const setFakeCookie = async (app, getPath) => {
+  let cookies;
+  await request(app)
+    .get(getPath)
+    .set('Cookie', ['ddd=eee'])
```

Comment:
can we use something else here rather than `ddd=eee`

Reply:
yes (but it makes no difference)

Reply:
I know but call it something like `someFakeCookie` so it reads a little better.